calitp-data/calitp_data/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "2022.2.9a0"
```

Comment:
Should this be updated to a `2023` version?

Reply:
Ah yeah this is a poetry thing and not actually used, will fix.